sdk/typescript/introspector/test/scan.spec.ts

```diff
@@ -171,13 +172,13 @@ describe("scan static TypeScript", function () {
 
     const result = scan(files)
     const expected: ScanResult = {
```

Comment:
I hope you're not managing this by hand 😱

Reply:
I do :)